src/components/PageHeader/AudioSetting.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Input, Text, useMatchBreakpoints } from '@pancakeswap-libs/uikit'
+import { useAudioModeManager } from 'state/user/hooks'
+import PancakeToggle from '../PancakeToggle/PancakeToggle'
+
+const StyledAudioSetting = styled.div`
+  margin-bottom: 16px;
+`
+
+const Label = styled.div`
+  align-items: center;
+  display: flex;
+  margin-bottom: 8px;
+`
+
+const Field = styled.div`
+  align-items: center;
+  display: inline-flex;
+
+  & > ${Input} {
+    max-width: 100px;
+  }
+
+  & > ${Text} {
+    font-size: 14px;
+    margin-left: 8px;
+  }
+`
+
+type AudioSettingModalProps = {
+  translateString: (translationId: number, fallback: string) => string
+}
+
+const AudioSetting = ({ translateString }: AudioSettingModalProps) => {
+  const { isSm, isXs } = useMatchBreakpoints()
+  const [audioPlay, toggleSetAudioMode] = useAudioModeManager()
+
+  return (
+    <StyledAudioSetting>
+      <Label>
+        <Text style={{ fontWeight: 600 }}>{translateString(999, 'Audio')}</Text>
```

Comment:
Remove the style and use the `bold` props

Reply:
Fixed here and in neighbouring settings modal components 👍